test/common/http/header_map_impl_test.cc

```diff
@@ -470,11 +478,9 @@ TEST_P(HeaderMapImplTest, InlineAppend) {
     EXPECT_EQ(headers.getViaValue(), "2.0 override");
   }
   {
-    // Set and then append. This mimics how GrpcTimeout is set.
+    // Set a string. This mimics how GrpcTimeout is set in Grpc::Common::toGrpcTimeout.
     TestRequestHeaderMapImpl headers;
-    headers.setGrpcTimeout(42);
-    EXPECT_EQ(headers.getGrpcTimeoutValue(), "42");
-    headers.appendGrpcTimeout("s", "");
+    headers.setGrpcTimeout("42s");
```

Comment:
I guess I would have kept this test the same, line for line, except use `"42"` in the first call to set.

Previously this stanza was testing the append path and now it isn't. It may also be that this path is fully covered in other stanzas in which case you could arguably just delete it.

Reply:
Will delete. 
I changed it because the append path isn't used for setting grpc-timeout anymore, but seeing as the behavior is already covered in the test with `appendVia`, it should just be removed.